lib/sprockets/sassc_processor.rb

```diff
@@ -0,0 +1,77 @@
+require 'sprockets/sassc_importer'
+require 'sprockets/sass_processor'
+
+module Sprockets
+  class SasscProcessor < SassProcessor
+    def initialize(options = {}, &block)
+      @cache_version = options[:cache_version]
+      @cache_key = "#{self.class.name}:#{VERSION}:#{Autoload::SassC::VERSION}:#{@cache_version}".freeze
+      @importer_class = options[:importer] || SasscImporter
+      @functions = Module.new do
+        include SassProcessor::Functions
+        include Functions
+        include options[:functions] if options[:functions]
+        class_eval(&block) if block_given?
```

Comment:
Since we name the `&block` method param, Ruby is going to turn the block into a Proc object.

Do you know if the garbage collector is going to get rid of the Proc object after the `Module.new do … end` block?

Not a big deal here, but this (Procs that get brought into closures) is one of the most common ways Ruby code creates memory leaks.